web-local/src/common/utils/incrementName.ts

```diff
@@ -23,3 +23,13 @@ export function getName(name: string, others: string[]) {
 
   return result;
 }
+
+export function getNextModelName(existingNames: string[]) {
+  let i = 1;
+  let name = `model_${i}`;
+  while (existingNames.includes(name)) {
+    i++;
+    name = `model_${i}`;
+  }
+  return name;
+}
```

Comment:
Nit: This method might be redundant. https://github.com/rilldata/rill-developer/blob/e5badfeb26299176958cedae2baf8f040908dcaf/web-local/src/common/utils/incrementName.ts#L13 
can be used with something like `getName("model", existingNames)`